pulsar-functions/java-examples/src/main/java/org/apache/pulsar/functions/api/examples/test/ByteBufferSerDe.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2018 Streamlio Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.pulsar.functions.api.examples.test;
```

Comment:
all these files should be under `org.apache.pulsar.functions.api.examples`, `test` is not needed.

Reply:
not sure about that, some of them are negative test function they are expected to fail, these should be kept in a separate package.

Reply:
I can see only one function is for negative test. It can be under `test` package. However all the other classes are examples:

- examples for customizing serde
- examples for using custom objects
- examples for using context.publish
- examples for windowing
- examples for config